### PR TITLE
Collect logs

### PR DESCRIPTION
**Pull request by carolinaborim**
Here @waldemarnt and I added Bunyan to write logs as JSON. 
In the first time we created a folder logs/metrics, but we figured out that we need a default log folder, so we added it too.
Besides of that, we also added a validation to verify if there is a authorization header.

### CODE REVIEW

**Comment by glauco** [comment]
LGTM!

File: src/app.js [hunk 1]
```diff
@@ -13,6 +13,14 @@ import UserInfoTransformer from './metrics/transformers/userInfoTransformer';
 import ResponseTimeTransformer from './metrics/transformers/responseTimeTransformer';
 import ResponseTimeExtractor from './metrics/responseTimeExtractor';
 
+const log = bunyan.createLogger({
+  name: 'fuse-equipment-api',
+  streams: [{
+    level: 'info',
```
Comment (andressa) [inline src/app.js]: Would it be useful to use level as 'debug' on a development environment and a 'info' level on a production? Does it make sense?

Comment (waldemarnt) [inline src/app.js]: The level means which kind of log we are writing. Metrics are not a **debug** in my point of view.

**Comment by lucaslago** [comment]
👍

File: src/app.js (hunk 1 above)
Comment (andressa) [inline src/app.js]: Ok. By now, as it is a PoC, I believe we can proceed like this. I would just like to point that in the future, it is not scalable to have multiple log files. =)

By that, I mean, LGTM 👍  =)

**Pull request merged**